#!/usr/bin/python3

def raise_exception_msg(message=""):
    """function raises a name exception"""

    raise NameError(message)
